src/test/groovy/com/github/vkorobkov/jfixtures/processor/ProcessorTest.groovy

```diff
@@ -377,9 +378,17 @@ class ProcessorTest extends Specification implements YamlVirtualDirectory {
 
     List<Instruction> load(String ymlFile) {
         def path = unpackYamlToTempDirectory(ymlFile) as String
-        def config = ConfigLoaderOld.load(path)
+        def config = loadConfig(path)
         def fixtures = new DirectoryLoader(path).load()
         new Processor(fixtures, config).process()
     }
 
+    private Root loadConfig(path) {
+        def loader = new ConfigLoader()
+        try {
+            loader.load("${path}/.conf.yml")
+        } catch(_) {
```

Comment:
Do we need to write to logs this exception?

Reply:
The issue is that we have a valid tests without existing config file at all, and in this case we want to pass the default empty config to processor.

Maybe it is better to replace try/catch with just checking the config file existence, what do you thing? So it would look like
```java
if (Files.exists(config) {
  loader.load(config)
} else {
  loader. defaultConfig();
}
```
What do you think?

Reply:
Yes, looks better. Can we use this variant:
```
if (Files.exists(config) {
  loader.load(config)
} 

loader.defaultConfig();
```